src/test/test-ns.c

```diff
@@ -47,6 +47,8 @@ int main(int argc, char *argv[]) {
         };
 
         static const NameSpaceInfo ns_info = {
+                .tmp = (char *) "/tmp/systemd-private-XXXXXX",
+                .var = (char *) "/var/tmp/systemd-private-XXXXXX",
```

Comment:
Sorry about that, this one sneaked here from another testing branch... I removed this last patch.